packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -84,9 +84,11 @@ const commands = [
         type: String,
         usage: 'serve [options]',
         description: 'Run the webpack Dev Server',
-        flags: devServer.reduce((cliArgs, { name, type, describe }) => {
-            return [...cliArgs, { name, type, description: describe }];
-        }, []),
+        flags:
+            devServerFlags &&
+            devServerFlags.reduce((cliArgs, { name, type, describe }) => {
+                return [...cliArgs, { name, type, description: describe }];
+            }, []),
```

Comment:
Don't forget `serve` should support `--mode`/`--devtool`/etc flags, all flags without serve